spec/requests/api/logging_spec.rb

```diff
@@ -0,0 +1,51 @@
+#
+# REST API Logging Tests
+#
+describe ApiController do
+  describe "Successful Requests logging" do
+    LOG_REQUESTS = {
```

Comment:
Perhaps `EXPECTED_LOGGED_PARAMETERS` or similar?

Just wondering, if these are important enough to encapsulate here in a constant, could they be better encapsulated in the logging code itself?